default.vcl

```diff
@@ -26,6 +26,9 @@ sub vcl_recv {
     elif (req.url ~ "^\/content\/feed\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}.*$") {
         set req.backend_hint = dynBackend.backend("ccf-content-rss");
     }
+    elif(req.url ~ "^\/content\/advanced-search.*$") {
+        set req.backend_hint = dynBackend.backend("combined-content-search")
+    }
```

Comment:
This will hit a `/content/advanced-search` endpoint of the `combined-content-search` service but that endpoint does not exist currently.
Do we plan to change the `/search` endpoint to `/content/advanced-search`?
If not, we should rewrite the URL here.